edgelet/edgelet-test-utils/src/lib.rs

```diff
@@ -9,29 +9,8 @@
     clippy::use_self
 )]
 
-extern crate chrono;
-extern crate edgelet_core;
 #[macro_use]
```

Comment:
Was this intentionally left behind?

Reply:
That's really weird that it built with those in place...

Reply:
Good catch!